selfdrive/car/tesla/values.py

```diff
@@ -21,6 +21,9 @@ class CAR:
     {
       1: 8, 3: 8, 14: 8, 21: 4, 69: 8, 109: 4, 257: 3, 264: 8, 267: 5, 277: 6, 280: 6, 283: 5, 293: 4, 296: 4, 309: 5, 325: 8, 328: 5, 336: 8, 341: 8, 357: 8, 360: 7, 373: 8, 389: 8, 390: 7, 415: 8, 513: 5, 516: 8, 518: 8, 520: 4, 522: 8, 523: 7, 524: 8, 525: 4, 526: 8, 527: 8, 529: 8, 532: 3, 536: 8, 537: 3, 539: 8, 542: 8, 545: 8, 546: 2, 551: 5, 552: 2, 556: 8, 558: 8, 568: 8, 569: 8, 574: 8, 577: 8, 582: 5, 583: 8, 584: 4, 585: 8, 590: 8,         606: 8,         622: 8, 627: 6, 638: 8, 641: 8, 643: 8, 660: 3, 665: 8, 692: 8, 693: 8, 695: 8, 696: 8, 697: 8, 699: 8, 700: 8, 701: 8, 702: 8, 703: 8, 704: 8, 708: 8, 709: 8, 710: 8, 711: 8, 712: 8, 728: 8, 744: 8,         760: 8, 771: 2, 772: 8, 775: 8, 776: 8, 777: 8, 778: 8, 780: 2, 781: 8, 782: 8, 783: 8, 785: 8, 787: 8, 788: 8, 791: 8, 792: 8, 796: 2, 797: 8, 798: 6, 799: 8, 804: 8, 805: 8, 807: 8, 808: 1, 809: 8,         812: 8, 813: 8, 814: 5, 815: 8, 820: 8, 823: 8, 824: 8, 829: 8, 830: 5, 831: 8, 836: 8, 840: 8, 841: 8, 843: 8, 845: 8, 846: 5,         849: 8, 852: 8, 853: 8, 856: 4, 857: 5, 859: 8, 861: 8, 862: 5, 863: 8, 872: 8, 873: 8, 875: 8,         877: 8, 878: 8, 879: 8, 880: 8, 882: 8, 884: 8, 888: 8, 889: 8, 893: 8, 894: 8, 896: 8, 901: 6, 904: 3, 905: 8,         907: 8, 908: 2, 909: 8, 910: 8, 912: 8, 920: 8, 921: 8, 925: 4, 926: 6, 936: 8, 937: 8, 941: 8, 949: 8, 952: 8, 953: 6, 957: 8, 968: 8,         970: 8,         973: 8, 977: 8, 984: 8, 986: 8, 987: 8, 989: 8, 990: 8, 1000: 8, 1001: 8, 1006: 8,                                              1016: 8,                   1019: 5,          1026: 8, 1028: 8, 1029: 8, 1030: 8, 1032: 1, 1033: 1, 1034: 8, 1035: 6, 1048: 1,                   1064: 8,          1070: 8, 1080: 8,                                              1160: 4,          1281: 8, 1291: 8, 1307: 8, 1328: 8, 1329: 8, 1332: 8, 1335: 8, 1337: 8, 1339: 2,          1361: 6, 1362: 6, 1368: 8, 1412: 8,                   1436: 8,          1456: 8, 1463: 8, 1465: 8, 1469: 8, 1476: 8,          1486: 8, 1497: 8,          1519: 8, 1521: 8, 1524: 8, 1527: 8, 1593: 8, 1601: 8, 1605: 8, 1609: 8, 1610: 2, 1611: 8, 1614: 8, 1617: 8, 1621: 8, 1625: 8, 1627: 8, 1630: 8, 1800: 4, 1804: 8, 1812: 8, 1815: 8, 1816: 8, 1824: 8, 1828: 8, 1831: 8, 1832: 8, 1840: 8, 1848: 8, 1864: 8, 1880: 8, 1892: 8, 1896: 8, 1912: 8, 1960: 8, 1992: 8, 2008: 3, 2043: 5, 2045: 4
     },
+    {
```

Comment:
this is just a subset of the original fingerprint, but 660 and 857 have different lengths

Reply:
Nope, I saw that and I have tried to figure out why that is for a week now. It might be a EU version?!

Reply:
are these values taken from another car besides your own?

Reply:
No these new ones are from mine, the ones before where borrowed from tinkla

Reply:
should I just replace?

Reply:
yeah, let's just replace with yours